dark/assets/css/styles.css

```diff
@@ -1051,6 +1051,11 @@ html {
   z-index: -1;
 }
 
+#contact center {

```

Comment:
Add a `padding-top: 30;` property to the css file @Rishav1707 

![image](https://user-images.githubusercontent.com/97069900/208220001-ca86da74-197b-4ca5-89cf-f0ac9f71c2b9.png)